### ISSUE TITLE
please add a Migration Guide from v3

### ISSUE THREAD

**Issue by xmasuhai** [prazdevs/pinia-plugin-persistedstate#335]
### Clear and concise description of the problem

 I was originally using version v3, but now when I upgraded to version v4, I encountered an error during project packaging: Property 'xxx' does not exist on type '_StoreWithState<"xxx".... After carefully reviewing the API documentation, I realized that the path used in version v3 has been deprecated and has been replaced by the pick in the current v4 version.

### Suggested solution

Hope to add a chapter on _Migration Guide from v3_ in the documentation.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by prazdevs** [comment]
That can be an idea to add some migration, as for now, you can refer to the changelog and breaking changes if you need it 

https://github.com/prazdevs/pinia-plugin-persistedstate/blob/main/CHANGELOG.md#v400

**Comment by prazdevs** [comment]
Closing this, as it's probably too late.

TL;DR for people still migrating (as of 4.2.0):
- No more global `beforeRestore` and `afterRestore` options.
- `beforeRestore` and `afterRestore` options are now called `beforeHydrate` and `afterHydrate`
- `paths` option is now called `pick`
- Nuxt auto-imported storage object is now called `piniaPluginPersistedstate`
- Nuxt config key is now called `piniaPluginPersistedstate`
